oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleAction.java

```diff
@@ -51,11 +51,19 @@ public OracleAction(OracleActionConfig oracleActionConfig) {
     @Nullable
     public Integer defaultBatchValue;
 
+    @Name("connectionType")
+    @Description("SID or Service Name")
```

Comment:
Whether to use an SID or Service Name when connecting to the database.

Reply:
added